Bit-Manipulation/test/GenerateSubSets.test.js

```diff
@@ -1,3 +1,4 @@
+import { expect } from 'vitest'
```

Comment:
I don't think you need to import `expect`. You should also test the error cases (at least the `RangeError`) btw.